charts/amazon-cloudwatch-observability/templates/certmanager.yaml

```diff
@@ -55,6 +55,9 @@ metadata:
   namespace: {{ .Release.Namespace }}
 spec:
   dnsNames:
+    {{- range $i, $customAgent := .Values.agents }}
+    - {{( printf "%s-target-allocator-service" $customAgent.name )}}
```

Comment:
Can we use a single DNS name for all the agent workloads ? Why do we need to create separate DNS names for every agent workload ?

Reply:
Spoke to @okankoAMZ offline, the helm-chart will support multiple target-allocators and hence will use DNS names for every workload